internal/database/recent_view_signal.go

```diff
@@ -29,9 +29,10 @@ type RecentViewSignalStore interface {
 }
 
 type ListRecentViewSignalOpts struct {
-	ViewerID int
-	RepoID   api.RepoID
-	Path     string
+	ViewerID    int
```

Comment:
nit: maybe make it clear this is a user id?
```suggestion
	ViewerUserID    int
```